homeassistant/components/update/significant_change.py

```diff
@@ -0,0 +1,30 @@
+"""Helper to test significant update state changes."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.core import HomeAssistant, callback
+
+from .const import ATTR_CURRENT_VERSION, ATTR_LATEST_VERSION
+
+
+@callback
+def async_check_significant_change(
```

Comment:
Is `significant` needed in name of module, method, etc.? What I'm wondering is do I have a `non-significant` change?